src/test/java/hudson/plugins/parameterizedtrigger/test/BuildTriggerConfigTest.java

```diff
@@ -121,6 +126,64 @@ public void testGetProjectListStatic() throws Exception {
 
     }
 
+    public void testGetProjectListWithWorkflow() throws Exception {
+        Project<?, ?> masterProject = createFreeStyleProject("project");
+        WorkflowJob p = jenkins.createProject(WorkflowJob.class, "workflowproject");
+        p.setDefinition(new CpsFlowDefinition("println('hello')"));
```

Comment:
nit: should this not be echo - so it uses workflow rather than just some groovy?